feat(apps): add user param to update app call (#32719)
RocketChat/Rocket.Chat: .changeset/witty-bats-develop.md

---
"@rocket.chat/meteor": patch
"@rocket.chat/apps": patch
"@rocket.chat/core-services": patch
"@rocket.chat/core-typings": patch
"@rocket.chat/fuselage-ui-kit": patch
"@rocket.chat/rest-typings": patch
"@rocket.chat/ddp-streamer": patch
"@rocket.chat/presence": patch
"rocketchat-services": patch
---

> Pass `user` param to apps-engine during app update, allowing apps' new `onUpdate` hook to know who triggered the update.

Reviewer: ```suggestion
Added the `user` param to apps-engine update method call, allowing apps' new `onUpdate` hook to know who triggered the update.
```

> Pass `user` param to apps engine during app update. This allows app engine's new `onUpdate` hook tou see the user performing the action.

Reviewer: ```suggestion
Pass `user` param to apps-engine during app update, allowing apps' new `onUpdate` hook to know who triggered the update.
```
